pom.xml

```diff
@@ -35,12 +35,12 @@
         <!-- Build properties (overridable in profiles) -->
         <enforcer.pluginVersions.banSnapshots>false</enforcer.pluginVersions.banSnapshots>
 
-        <spring-boot.version>1.5.17.RELEASE</spring-boot.version>
+        <spring-boot.version>1.5.18.RELEASE</spring-boot.version>
         <version.camel>2.21.0.fuse-740037</version.camel>
         <version.camel.extra>2.21.0.fuse-740034</version.camel.extra>
         <version.cxf>3.2.7.fuse-740024</version.cxf>
         <version.fabric8>3.0.11.fuse-740027</version.fabric8>
-        <version.fabric8.maven.plugin>3.5.33.fuse-740027</version.fabric8.maven.plugin>
```

Comment:
@orpiske No way to find this dependency, I bumped to nearer one. Please let me know if this generates any issue.

Reply:
I think we could already be using the 750xx deps, but as you suggested we can split this in two parts. So, it LTGM.